### ISSUE TITLE
[GENERAL] Missing localizations: SL (Slovenian), LLD (Ladin)

### ISSUE THREAD

**Issue by abaumg** [immuni-app/immuni-documentation#95]
# Issue content
Apart from French (in the Aosta Valley) and German (in South Tyrol), also Slovene (in Trieste, Gorizia and Udine) and Ladin (in South Tyrol and Trentino) are co-official languages in Italy.

Currently, only the French and German localizations are included. The Slovene and Ladin localizations are missing.
